ground/gcs/gcs.pri

```diff
@@ -67,6 +67,7 @@ macx {
     GCS_DATA_BASENAME = Resources
     GCS_DOC_PATH     = $$GCS_DATA_PATH/doc
     GCS_BIN_PATH     = $$GCS_APP_PATH/$${GCS_APP_TARGET}.app/Contents/MacOS
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9

```

Comment:
(So it breaks the build if you don't have this, because xcode-utils dynamic links against the version of the libraries that matches your deployment target).